### PR TITLE
Add Trunk & Tidbits for May 2024. Depends on contrib guide.

### PR DESCRIPTION
**Pull request by andypiper**
Signed-off-by: Andy Piper <andypiper@users.noreply.github.com>

### CODE REVIEW

**Comment by andypiper** [comment]
Do not merge without deciding on merging https://github.com/mastodon/.github/blob/add-contrib/CONTRIBUTING.md

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+We’re continuing to drive towards Mastodon 4.3, our next major release.
+
+The changes below are available to users of instances that are running a recent “nightly” version (for example, [mastodon.social](http://mastodon.social) or [mastodon.online](https://mastodon.online)), and will come to others as part of the 4.3 release.
+
+# Web app and Backend
+
+<div class="features-list">
+
+- {{< feature-label name="interface" >}} Allow admins to configure the instance favicon and logo. {{< github-pr id=30040 authors="FawazFarid" >}}
+- {{< feature-label name="new-feature" >}} Add OpenTelemetry instrumentation. This allows admins to send traces to their own OpenTelemetry server, allowing monitoring of backend errors and performance with much more details than logs - useful to spot performance issues and fix them. {{< github-pr id=30130 authors="robbkidd, julianocosta89, renchap" >}}
```
Comment (renchap) [inline content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:39]: Shold be `name="new feature"`, with a space (same for multiple others below)

Comment (renchap) [inline content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:39]: There should be no space after commas in the authors list

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- {{< feature-label name="security" >}} A pair of OAuth-related updates:
```
Comment (renchap) [inline content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:40]: Oauth changes are not really security, but maybe we need an "authentication" label here?

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  - Implement RFC 8414 for OAuth 2.0 server metadata {{< github-pr id=29191 authors="thisismissem" >}}
```
Comment (renchap) [inline content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:41]: Those are not rendered as a sub-list

image

I am not sure if this comes from my custom CSS. Maybe we could put the 2 oauth changes on 2 lines, and not try to group them?

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md (hunk 1 above)
Comment (andypiper) [reply content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:39]: oops, thanks, on it.

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md (hunk 2 above)
Comment (andypiper) [reply content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:40]: Fair point, or API here?

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md (hunk 3 above)
Comment (andypiper) [reply content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:41]: ok

File: content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md (hunk 1 above)
Comment (andypiper) [reply content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md:39]: hmm, no, this is correct - the name is `new-feature` and it renders as "new feature" 👍🏻

**Comment by andypiper** [comment]
Addressed review comments and also fixed the heading hierarchy. Improved a few other small phrases.

**Review by renchap — APPROVED** [review]

**Pull request merged**